src/lidar/lidar_metadata_in_db.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct  5 16:34:48 2021.
+
+@author: pkh35
+"""
+
+import geoapis.lidar
+import json
+import geopandas as gpd
+import os
+import pandas as pd
+import zipfile
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+from geoalchemy2 import Geometry
+from sqlalchemy.orm import sessionmaker
+
+
+Base = declarative_base()
+
+
+class Lidar(Base):
+    """Create lidar table in the database."""
+
+    __tablename__ = 'lidar'
+    unique_id = Column(Integer, primary_key=True, autoincrement=True)
+    filepath = Column(String)
+    Filename = Column(String)
+    filename_no_format = Column(String)
+    geometry = Column(Geometry('POLYGON'))
+
+
+def get_lidar_data(file_path_to_store, geometry_df):
+    """To get the LiDAR data from opentopography using geoapis.
+
+    https://github.com/niwa/geoapis
+    """
+    geometry_df.set_crs(crs='epsg:2193', inplace=True)
+    lidar_fetcher = geoapis.lidar.OpenTopography(cache_path=file_path_to_store,
+                                                 search_polygon=geometry_df, verbose=True)
+    lidar_fetcher.run()
+
+
+def get_files(filetype, file_path_to_store):
+    """To get the path of the downloaded point cloud files."""
+    file_list = []
+    files_path = []
+    for (paths, dirs, files) in os.walk(file_path_to_store):
+        for file in files:
+            if file.endswith(filetype):
+                file_list.append(os.path.join(paths, file))
```

Comment:
How about this change instead? Paths-> path and dirs -> dir
```suggestion
    for (path, _dirs, files) in os.walk(file_path_to_store):
        for file in files:
            if file.endswith(filetype):
                file_list.append(os.path.join(path, file))
```